pages.ko/common/banner.md

```diff
@@ -1,15 +1,15 @@
-# banner
-
-> 주어진 인자를 큰 ASCII art로 출력.
-
-- 텍스트 메시지를 큰 배너로 출력(따옴표는 선택 사항):

```

Comment:
Why this text was changed?

Reply:
wow, I have no idea! those changes are not caught in my IDE, when compared to master

Reply:
Looks like you have added an invisible char, but Idk what.
![Screenshot_20201003_161908](https://user-images.githubusercontent.com/1536740/95000092-30629d80-0594-11eb-8acb-6e98389f5e70.png)

Reply:
`^M` is the [carriage return character](https://en.wikipedia.org/wiki/Carriage_return) (the CR in CRLF), usually represented as `\r`. You might want to make sure your editor (or [git](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_autocrlf)) is not configured to make changes to them. Usually only the line feed character, `\n`, is expected as a line terminator.

Reply:
I know this is old, but I'd like to say (for history's sake), I fixed all of this in #5136.  It looks like @Proscream commited a few CRLF files and those were always revert when anyone tried to modify the files.  I'll resolve this convo now too.